mapie/tests/test_quantile_regression.py

```diff
@@ -320,18 +354,29 @@ def test_wrong_alphas_types(alphas: float) -> None:
         match=r".*Invalid alpha. Allowed values are float.*",
     ):
         mapie_reg = MapieQuantileRegressor(alpha=alphas)
-        mapie_reg.fit(X_train, y_train, X_calib, y_calib)
+        mapie_reg.fit(X_train, y_train, X_calib=X_calib, y_calib=y_calib)
+
+
+def test_alpha_in_predict() -> None:
+    """Checking for alpha in predict"""
```

Comment:
Checking for alpha in predict raises a warning/error.

Reply:
done